tests/unit/dataactvalidator/test_c14_award_financial.py

```diff
@@ -0,0 +1,33 @@
+from tests.unit.dataactcore.factories.staging import AwardFinancialFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+_FILE = 'c14_award_financial'
+
+def test_column_headers(database):
+    expected_subset = {'row_number', "fain", "uri", "piid"}
+    actual = set(query_columns(_FILE, database))
+    assert (actual & expected_subset) == expected_subset
+
+
+def test_success(database):
+    """ Test cases with different combinations of fain, uri, and piid """
+
+    # Test with only one present
+    award_fin_fain = AwardFinancialFactory(uri = None, piid = None)
+    award_fin_uri = AwardFinancialFactory(fain = None, piid = None)
+    award_fin_piid = AwardFinancialFactory(fain = None, uri = None)
+    # Test with all three
+    award_fin = AwardFinancialFactory()
+    # Test with one missing
+    award_fin_no_fain = AwardFinancialFactory(fain = None)
```

Comment:
When we do the updates from the validation rules sheet, I'm thinking we'll probably add a c14_award_financial_2 file that will catch that error, which would then also get its own unit test.